x-pack/test/security_solution_cypress/cypress/README.md

```diff
@@ -44,10 +44,11 @@ Please, before opening a PR with the new test, please make sure that the test fa
 
 Note that we use tags in order to select which tests we want to execute:
 
-- `@serverless` includes a test in the Serverless test suite for PRs (the so-called first quality gate) and QA environemnt (the so-called second quality gate). You need to explicitly add this tag to any test you want to run in CI for serverless. 
+- `@serverless` includes a test in the Serverless test suite for PRs (the so-called first quality gate) and QA environment for the periodic pipeline. You need to explicitly add this tag to any test you want to run in CI for serverless. 
+- `@serverlessQA` includes a test in the Serverless test suite for the Kibana release process of serverless. You need to explicitly add this tag to any test you want yo run in CI for the second quality gate.
```

Comment:
Could we have only `@serverless` and `@broken...` tags without `@serverlessQA`?

My concern here is that each test may have a lot of tags which increases cognitive load. To simplify the process we could simplify tagging and for example run a test everywhere until `@brokenIn...` or `@skipIn...` tag is specified.

Reply:
For MKI projects will have 2 different executions:
- Periodic pipeline where we are going to execute all the tests.
- The kibana release process where just a subset of the above are going to be executed.

If we are going to the broken or skip we should differentiate between the periodic pipeline and the release process what is going to increase the number of tags as well.

The `@serverlessQA` was added after a conversation with @yctercero to simplify the process of enabling tests on the Kibana second quality gate and at the same time give visibility to the ones that are executed there since are going to be a subset.

Thoughts @maximpn @yctercero?

Reply:
@maximpn totally understand where you're coming from. Having a lot of tags can increase cognitive load for the developer. Having only a subset of test running in MKI came out of a discussion we had with @banderror . Any failures in the MKI environment would block serverless release. There are certain tests where if they fail, we can fix forward, and some that should absolutely block release. 

Is there an in-between that could work? What if we have:

- `ess` - runs in ESS env
- `serverless` - runs in serverles env and periodic pipeline (failures don't block release)
- `serverlessQA` - runs in kibana release process (failures block release)
- `skipInEss` - skipped for ESS env
- `skipInServerless` - skipped for all serverless related environments

With the expectation that when adding a `skip...` tag, an issue should be added in the comments with details as to why it is skipped (flakey, bug, etc.).

Reply:
@yctercero yes, it makes total sense 👍 

So we get rid of `@broken...` tags in favor of `@skip...` tags and use `@serverlessQA` for a limited set of tests due to reasons like cost restrictions. Correct me if I'm wrong here.

This approach will allow us to implement automatic test skipping via tags at some point.

> Having only a subset of test running in MKI came out of a discussion

I'd be great to have an explanation in README why we need `@serverlessQA` and what judgment should be used to mark a test with this tag.

Reply:
> @MadameSheema do you think we can use it instead of the one suggested in the PR description?

Absolutely!! :)

Reply:
> @MadameSheema Do we have an epic for the whole work of running tests against MKI?

@banderror I've that ticket but I need to complete it with all the information I got lately https://github.com/elastic/kibana/issues/173327

Reply:
> It will be up to each team to decide what tests they will consider critical enough to be executed in QA. Basically:
- They should be stable, otherwise if they are flaky, they will be blocking the release pipeline.
- They should be critical enough, so that when they fail, there's a high chance that it indicates a critical bug in Kibana => that's why the whole release process should be blocked.

Having something like that in README would be perfect.

Reply:
I added that info to the readme, in parallel I'm working as well in extended documentation around automation.